test/screens_web/views/v2/audio/elevator_status_view_test.exs

```diff
@@ -0,0 +1,77 @@
+defmodule ScreensWeb.V2.Audio.ElevatorStatusViewTest do
+  use ScreensWeb.ConnCase, async: true
+  alias ScreensWeb.V2.Audio.ElevatorStatusView
+
+  describe "No elevator alerts" do
+    test "renders all clear message" do
+      assigns = %{
+        active_at_home_pages: [],
+        list_pages: [],
+        upcoming_at_home_pages: [],
+        elsewhere_pages: []
+      }
+
+      assert render(assigns) =~ "All elevators are working at this station."
+    end
+  end
+
+  describe "Single elevator alert" do
+    test "renders no additional closures messages" do
+      assigns = %{
+        active_at_home_pages: test_stations(),
+        list_pages: [],
+        upcoming_at_home_pages: [],
+        elsewhere_pages: []
+      }
+
+      assert render(assigns) =~
```

Comment:
I think this and the assert in the above test case should be an exact match check with `==`

Reply:
Because the strings have a lot of garbage characters I opted to just do multiple `=~` asserts on the important substrings for the tests, to make it more readable.